### DISCUSSION TITLE
Balkonkraftwerk an Senec hybrid V3

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#2070] [General]
folgende Situation
seit 9 Monaten läuft eine PV-Anlage 11,745kWp mit dem Senec Speicher.
Dieser ist bekannterweise ein zwei Phasen Einspeißer. Die dritte Phase wird vom EnFluRi immer mit einem positivem Wert angezeigt.
Über den Zähler wird dieser aber verrechnet.
Nun habe ich ein Balkonkraftwerk (BKKW) an dieser dritten Phase angeschlossen und siehe da der EnFluRi zeigt je nach Leistung des BKKW 
auch einen negativen Wert an, also wird auch hier eingespeißt.
Das Dashboard von deye zeigt mir weder einen Stromfluß ins Haus noch einen ins Netz. Es wird nur produziert.
Jetzt die Masterfrage: Wenn der EnFluRi mitbekommt dass hier auf der (senec)freien Phase eingespeißt wird,
fließt das in die Anzeige von Solectrus mit ein ???
Wäre über Rückmeldungen eines solchen Konzepts dankbar.
LG i n die Runde

**Comment by bennysweb** [comment]
Hi, ich habe das auch genau so gelöst, also Balkonkraftwerk an der dritten Phase. Senec registriert die Einspeisung davon als geringeren Hausverbrauch, dementsprechend kann der Hausverbrauch auch mal auf 0 sinken, oder sogar die Einspeisung höher sein als die Erzeugung der Senec-Anlage. Dies wird auch exakt so in Solectrus abgebildet. 
VG Benny

**Comment by Zeppelin-PV** [comment]
Danke für die Rückmeldung.
Das heißt mein Solectrus zeigt einfach den Betrag X vom BKKW in der Summe von der PV-Anlage mit an.
Dann brauche ich auch kein Dashboard von deye, den Wechselrichter stelle ich dann in der FritzBox auf lokal wegen dem nachHause telefonieren.
Ist eigentlich das Installationskennwort /Passwort des Senec Speichers überall gleich oder individuell ?
Würde zu gern die 70% öffnen

**Reply by bennysweb** [reply]
Nicht ganz, Solectrus zeigt weniger bis gar keinen Hausverbrauch an. Also nicht mehr Erzeugung, sondern weniger Hausverbrauch. Das Kennwort ist übrigens bei allen Anlagen gleich ;-)

**Reply by Zeppelin-PV** [reply]
... ich habe schon so viele Zugangsdaten bekommen, aber ohne Erfolg; wäre dankbar für jeden (funktionierenden) Hinweis

**Reply by bennysweb** [reply]
Meld dich mal per Mail: b.2801.k@web.de

**Comment by Zeppelin-PV** [comment]
Und schon geht's los. Obwohl die Module noch nicht an ihrem endgültigen Platz montiert sind.
Ein Minus an der dritten Phase erweckt positive Gefühle.
![grafik](https://github.com/solectrus/solectrus/assets/126392805/9038e779-1486-4b86-b083-b2710c717f03)

**Comment by Zeppelin-PV** [comment]
Nachdem nun das BKW an Ort und Stelle ist muss ich sagen die Idee ist gut und hat sich gelohnt
Der Hausverbrauch geht nochmals deutlich zurück und die Senec PV speist nahezu brutto zu netto
![grafik](https://github.com/solectrus/solectrus/assets/126392805/b0a6001c-41b7-454f-b468-244e08dda47b)
